podpac/core/compositor/tile_compositor.py

```diff
@@ -89,7 +89,7 @@ def tile_width(self):
 
 
 @common_doc(COMMON_DATA_DOC)
-class UniformTileMixin(podpac.Node):
+class UniformTileMixin(tl.HasTraits):
```

Comment:
What is the rule for `Mixins`? Is there any harm inheriting from `Node`? Why not inherit from `object` then? (probably need the `traitlets` metaclass functionality I'd reckon. )

Reply:
I don't know exactly. I think you're right, object is probably better. I'm not sure if the traits will work without subclassing tl.HasTraits here, but it might because it is always mixed with a different class that does inherit from tl.HasTraits.

Reply:
I don't remember why I changed it.

Reply:
I just did a quick test. Looks like inheriting from `object` works well. 

```python
import traitlets as tl
class Mix(object):
    l = tl.List()

class T(tl.HasTraits):
    u = tl.Unicode()

class G(Mix, T):
 pass

g = G()
g.l
[]
g.trait_names()
['l', 'u']
```

I don't think I'll change it... we could do a PR to change all Mixin's to `object`, but I'd rather understand it better before doing something like that. This was just for interest.